### PR TITLE
Added Recipe book

### PR DESCRIPTION
**Pull request by pavitraag**
Closes: #598 

- **Title** :  Recipe Book
- **Name:** : Pavitraa
- **Idenitfy yourself:** : GSSOC contributor 

### Screenshots 📷
image

### Type of change ☑️

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### Checklist: ☑️

- [X] My code follows the [Contributing Guidelines](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CONTRIBUTING.md) & [Code of Conduct](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CODE_OF_CONDUCT.md) of this project.
- [X] This PR does not contain plagiarized content.
- [X] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly wherever it was hard to understand.
- [X] My changes generate no new warnings.

### CODE REVIEW

File: Front-end-Projects/Basic/Recipe book/Readme.md
Comment (Avdhesh-Varshney) [inline Front-end-Projects/Basic/Recipe book/Readme.md:1]: properly fill the readme template

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]
Also read & follow the guidelines properly.

**Comment by pavitraag** [comment]
@Avdhesh-Varshney done with changes

**Comment by Avdhesh-Varshney** [comment]
Add a ss with format `webp` and named to be `screenshot`

**Comment by pavitraag** [comment]
@Avdhesh-Varshney done

**Comment by Avdhesh-Varshney** [comment]
Re-raise this pr. Closed because only due to 13 numbers of commits.

**Pull request closed without merging**